run/john.conf

```diff
@@ -361,11 +361,17 @@ Frequency = 141
 
 [ZTEX:sha512crypt]
 #TargetRounds = 5000
+# Design tools reported possible frequency to be 225.1 MHz.
+# On tested boards it often fail (with error or timeout), or doesn't
+# find passwords at such high frequency. Default frequency is set
+# to 135 MHz.
```

Comment:
Grammar fix: s/fail/fails/

More importantly, I think we never encountered a board where this worked at anywhere close to 225 MHz for a bitstream with all 10 units in it. The best I got out of it is 180 MHz or so with 5 units disabled in john.conf (and 5 left enabled). With all 10 units, the best realistic speed is 160 MHz. Perhaps we should give 160 MHz as an example in the comment, e.g. saying "Some lucky boards run this OK at 160 MHz, but usually not more."